site/app/models/OfficeHoursQueueModel.php

```diff
@@ -205,6 +212,14 @@ public function getCurrentQueueTimeIn() {
         return $this->current_queue_state['time_in'];
     }
 
+    public function getCurrentTimePaused() {
+        return $this->current_queue_state['time_paused'];
+    }
+
+    public function getCurrentTimePausedStart() {
```

Comment:
Not sure if this returns a string or `\DateTime`?

Reply:
Both of those functions are returning a string. I see how that may be misleading but I was trying to follow the naming scheme of the functions around it.

Reply:
Ah, alright, so we can append `: string` here as the return type hint.

Reply:
This change actually had to become `: ?string` as `time_paused_start` could be `NULL` which we had discussed about on another change above.